### DISCUSSION TITLE
GEVI Agents

### DISCUSSION THREAD

**Discussion by ms52538** [CodyBerenson/PGMA-Modernized#293] [General]
Hi, I am having difficulty matching up both scenes and feature films with the GEVI agents. 

Scene examples not matching:
(Raw Fuck Club) - Me As Top For The First Time - {184430} (2023) - [AsstonXXX,Simon Montana]
(Raw Fuck Club) - Mick Weston Fucks Alex Kof {191834} (2022)
(Raw Fuck Club) - Daddy Breeds Colby Keller w - Tucker Wilde - {185529} (2023)

Feature Film examples of not matching:
**(Naked Sword) - Global Entry - Portugal (2023)** - [Alex Ink,Alex Palmieri,Angelo Pedro,Gustavo Cruz,Jamal Costa,Javi Gray,Lobo Carreira,Rafael Ferreira]
**(Naked Sword) - Global Entry - Sicily (2023)** - [Alex Horney,Alex Palmieri,Davide Bubi,Giuspel,Johnny Viper,Justin Jett,Scott Carter]

Thanks!
M

**Comment by JPH71** [comment]
Fir the gevi scenes, you need to add an N, R or L to deal with rotating the
picture.

I will look at why the others don't scrape...

**Reply by ms52538** [reply]
Thank you!, totally forgot the N for the scene matching.
I'll wait to hear back from you on the films not matching. 

M
